bin/commands/start.py

```diff
@@ -0,0 +1,94 @@
+import os
+import re
+
+import utils
+
+DESCRIPTION = "Start a container from an existing dhis2-data Docker image or from an exported file"
+
+
+def setup(parser):
+    parser.add_argument(
+        "image_or_file", metavar="IMAGE_OR_EXPORT_FILE", help="Docker image or exported file"
+    )
+    utils.add_core_image_arg(parser)
+    parser.add_argument(
+        "-d", "--detach", action="store_true", help="Run containers on the background"
+    )
+    parser.add_argument(
+        "-k", "--keep-containers", action="store_true", help="Keep existing containers"
```

Comment:
Maybe this has been discussed previously but I wod expect this to be the default behavior and have a flasg to clean instead.

Reply:
Indeed, we discussed this with @ifoche, I was also hesitant. We finally decided to keep it this way since the main usage of this was as training.